playbooks/setup-jenkins-slave.yml

```diff
@@ -0,0 +1,69 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: False
+  tasks:
+    - name: Get facts about servers with a given name
+      local_action:
+          module: rax_facts
+          name: "{{ name }}"
+
+    - name: Show IP of server with given name, fail if no such server exists in region
+      debug: var=rax_accessipv4
+      failed_when: rax_accessipv4 is undefined
+
+    - name: Set ansible host to IP address of server with given name
+      set_fact:
+          ansible_host: "{{ rax_accessipv4 }}"
+
+    - name: Add servers to cloud_instances host group
+      local_action:
+        module: add_host
+        hostname: "{{ name }}"
+        ansible_host: "{{ rax_accessipv4 }}"
+        ansible_user: root
+        groupname: cloud_instances
+
+    - name: Wait for SSH to be available on host
+      wait_for: port=22 host="{{ ansible_host }}"
+
+
+- hosts: cloud_instances
+  tasks:
+    - name: Install apt packages
+      apt:
+        pkg: "{{ item }}"
+        state: installed
+        update_cache: yes
+      with_items:
+        - git
+        - default-jre
```

Comment:
If you mean default-jre specifically, it's needed to run the Jenkins slave jar since it's Java based.